src/app/upgrade.cpp

```diff
@@ -75,11 +75,27 @@ namespace
             , QLatin1String("Preferences/WebUI/HTTPS/KeyPath")
             , Utils::Fs::toNativePath(configPath + QLatin1String("WebUIPrivateKey.pem")));
     }
+
+    void removeUnusedOptions()
```

Comment:
I think this is a bad idea (and has been discussed many times), users won't able to downgrade to older qbt versions without losing some of the configurations.

Reply:
Fair enough. I at least updated the relevant wikipage. What about recycling this PR as a simple log "You are using deprecated option X. It can be safely removed from your config file. Check the wiki (wikipage url goes here) for potential replacements"?

Reply:
I would just do nothing with it.
These log messages will only be annoying, especially for those who have never touched the configuration file manually.

Reply:
Ok then, closing this.